src/app/customer-search/customer-search.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit } from '@angular/core';
+import { TestsDataService } from '../services/tests-data.service';
+import { TfiDataService } from '../services/tfi-data.service';
+import { ThsDataService } from '../services/ths-data.service';
+import { TsScreenerDataService } from '../services/ts-screener-data.service';
+
+@Component({
+  selector: 'customer-search',
+  templateUrl: './customer-search.component.html',
+  styleUrls: ['./customer-search.component.css']
+})
+export class CustomerSearchComponent implements OnInit {
+  public idSearch: string;
+  public searchBtn: boolean = true; // Search button is disabled while querying DB
+  public invalidID: boolean = false;
+  public resultsTable = [];
```

Comment:
```suggestion
  public resultsTable: AppointmentsReference = [];
```
We should add an AppointmentsReference Object to type check what angular is actually looking for. 
    Example: https://angular.io/guide/displaying-data#creating-a-class-for-the-data
_Its important to note that this class will also be what the API service will return (That way, we have very little to worry about regarding data conversion._

Reply:
We should (and will) discuss these data structures at this next team meeting.